presto-parser/src/main/java/com/facebook/presto/sql/tree/GroupingElement.java

```diff
@@ -31,4 +31,16 @@ protected GroupingElement(Optional<NodeLocation> location)
     {
         return visitor.visitGroupingElement(this, context);
     }
+
+    void validateExpressions(List<Expression> expressions)
+    {
+        expressions.forEach(expression -> ExpressionTreeRewriter.rewriteWith(new ExpressionRewriter<Void>()
```

Comment:
I see. So in order to visit an expression tree, we have to use a `ExpressionRewriter`?  (I searched and didn't find `ExpressionVisitor` 🤦‍♀️  )

Reply:
I didn't find it either... @highker might know?